content/broker/reference/rate-limits/_index.en.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Rate Limits
+linktitle: Rate Limits
+description: Altinn 3 Broker Capabilities and Features.
+tags: []
+toc: true
+weight: 40
+---
+
+## Rate limits
+
+Rate limit applies per subscription key (Ocp-Apim-Subscription-Key). Contact us for more capacity.
```

Comment:
Consider adding a single paragraph to justify the Rate Limits;
Something like "We have introduced the following rate limits to balance the resource usage per party and avoid overloading the infrastructure. Using Events/Webhooks in your solution should eliminate the need for polling many of these operations". With maybe a link to the Events page?